release_notes/Service_Fabric_ReleaseNotes_90CU9.md

```diff
@@ -0,0 +1,61 @@
+| **Windows - 9.0.1526.9590<br>Ubuntu 18 - 9.0.1463.1<br>Ubuntu 20 - 9.0.1463.1** | **Feature** | Key Value Store (KVS) | **Brief Description**:  KVS backed partitions, including Actors and System Services, experience quorum loss due to StoreLVIDLimitHit error code (0x80071d4d), indicating the depletion of values for a monotonically increasing ID field. This results in unavailability of the partition until a manual offline maintenance task is executed to resolve the issue. <br>**Solution**: To address the error, KVS replicas now restart automatically and initiate the maintenance task (Extensible Storage Engine database compaction) during the next open. While there may be temporary availability loss in partitions depending on the task duration, no manual intervention is required to mitigate the issue.
+| **Windows - 9.0.1526.9590<br>Ubuntu 18 - 9.0.1463.1<br>Ubuntu 20 - 9.0.1463.1** | **Feature** | Key Value Store (KVS) | **Brief Description**: Auto-compaction now emits health events at the start and completion, visible in the Service Fabric Explorer, enhancing its monitoring capabilities. These events complement trace data, providing effective visibility into active auto-compaction progress.
+
+## Retirement and Deprecation Path Callouts
+* As aligned with [Microsoft .NET and .NET Core - Microsoft Lifecycle | Microsoft Learn](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-and-net-core), SF Runtime has dropped support for Net Core 3.1 as of December 2022. For  supported versions see [Service Fabric supported Linux versions](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-versions#current-versions-1) and migrate applications as needed.
+* Ubuntu 18.04 LTS will reach its 5-year end-of-life window on June, 2023. Service Fabric runtime will drop support for 18.04 LTS after the published date, and we recommend moving your clusters and applications to supported versions listed here: [Service Fabric supported Linux versions](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-versions#supported-linux-versions-and-support-end-date)
+* Service Fabric runtime will soon stop supporting BinaryFormatter based remoting exception serialization by default and move to using Data Contract Serialization (DCS) based remoting exception serialization by default. Current applications using it will continue to work as-is, but Service Fabric strongly recommends moving to using Data Contract Serialization (DCS) based remoting exception instead.
+* Service Fabric runtime will soon be archiving and removing Service Fabric runtime version 6.4 packages and older, as well as SDK version 3.3 packages and older from the package Download Center. Archiving/Removing will affect application scaling and re-imaging of virtual machines in a Service Fabric Cluster running on unsupported versions. After older versions are removed/archived, this may cause failure while rolling back when the current in-progress upgrade has errors.
```

Comment:
just to be clear, we are leaving github repository with this code base available though? (seems kind of weird)